dagrt/builtins_python.py

```diff
@@ -85,6 +119,30 @@ def builtin_matmul(a, b, a_cols, b_cols):
     return res_mat.reshape(-1, order="F")
 
 
+def builtin_reshape(a, a_cols):
+    return a.reshape(-1, a_cols, order="F")
```

Comment:
Since dagrt arrays don't have shapes other than "1D vector", "reshape" is an odd addition IMO.

Reply:
This was only being used in one place in BDF, and I managed to remove it (562b2b5)